app/views/studies/sync_study.html.erb

```diff
@@ -183,45 +183,34 @@
 
     // pre-select all MM coordinate matrix & BAM associations, if present
     $(document).ready(function() {
-        var allMatrices = gatherFilesByType('MM Coordinate Matrix');
-        var allBams = gatherFilesByType('BAM');
         var disabledCss = {
             'pointer-events' : 'none'
         };
+
+        function appendOptionsAndSetDisabled(optionsArray, targetSelect, targetId, parentSelect) {
```

Comment:
Please add a brief comment above this function signature, to help lazy engineers like me :).

Reply:
In looking back at this function, I've realized that it in fact may not be needed at all, or at least most of what is happening doesn't need to.  I'll refactor and comment before getting further review.

Reply:
71d66e3